app/templates/admin/sales.hbs

```diff
@@ -1 +1,18 @@
-{{outlet}}
+<div class="ui grid">
+  <div class="row">
+    <div class="{{if device.isMobile 'sixteen' 'three'}} wide column tabbed-navigation">
```

Comment:
Remove the `tabbed-navigation` class. Doesn't make sense. 
And wrap the items below inside a `tabbed-navigation` component. Pass parameter `isVertical=true`